source/routing/raptor.cpp

```diff
@@ -360,7 +361,8 @@ RAPTOR::compute_all(const map_stop_point_duration& departures,
                     const std::vector<std::string>& forbidden_uri,
                     bool clockwise,
                     const boost::optional<navitia::time_duration>& direct_path_dur,
-                    const size_t max_extra_second_pass) {
+                    const size_t max_extra_second_pass,
+                    const navitia::time_duration& transfer_penalty) {
```

Comment:
It would modify some more lines as we do not care for transfer_penalty when calling compute_all so far and we would...